helloworldwebapp/templates/csr/config/staging/service.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: helloworldwebapp-staging
+
+---
+
+apiVersion: v1
+kind: Service
+metadata:
+  name: helloworldwebapp-service
+  namespace: helloworldwebapp-staging
+spec:
+  ports:
+  - protocol: TCP
+    port: 80
+  selector:
+    app: helloworldwebapp
```

Comment:
Is this intentional?

I end up with 2 applications (and 1 contains just the services):

![image](https://user-images.githubusercontent.com/8437718/59509421-f5a89380-8e7e-11e9-90e0-607de72bdcd2.png)

Reply:
Didn't realize these were grouped there, consolidated the labels/ app name.